tester_app/client/app/bundles/HelloWorld/startup/helloWorldAppClient.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import createStore from '../store/helloWorldStore';
+import HelloWorld from '../containers/HelloWorld';
+
+const helloWorldAppClient = props => {
+  const store = createStore(props);
+  const reactComponent = (
+    <Provider store={store}>
+      <HelloWorld />
+    </Provider>
+  );
+  return reactComponent;
+};
+
+// Export is needed for the hot reload server
+export default helloWorldAppClient;
```

Comment:
// Note we export as camelCase and not PascalCase because helloWorldAppClient is a generator function 
// rather than a class.